### DISCUSSION TITLE
bat for RHEL distro

### DISCUSSION THREAD

**Discussion by mdrijwan** [sharkdp/bat#3149] [Ideas]
is there any plan to release bat for RHEL8 pr 9?

**Comment by keith-hall** [comment]
The maintainers of `bat` are generally only responsible for making binary releases of `bat` itself, it would be impossible for us to package `bat` for every single distro - many of which we don't even use ourselves, and which happens outside this repository. If you want `bat` to be included in RHEL, your best bet is to either contact some RHEL package maintainer and ask them nicely, or work out how to do it yourself and make a PR to the relevant repo.
Otherwise, there's still the option of manually installing from a binary release, or installing via Cargo or building via source.

**Reply by blinxen** [reply]
`bat` should be available in both `epel8` and `epel9` repositories.
